.github/workflows/main.yml

```diff
@@ -255,6 +261,56 @@ jobs:
         run: |
           docker image ls --format "{{.Repository}}:{{.Tag}}" | grep "$DOCKER_IMG_NAME_LIBVCX" || { echo "Image $DOCKER_IMG_NAME_LIBVCX was not found!" ; exit 1; }
 
+  build-image-libvcx-tester:
+    needs: [workflow-setup, build-image-alpine-core]
+    runs-on: ubuntu-20.04
+    env:
+      DOCKER_BUILDKIT: 1
+      CACHE_KEY_LIBVCX_TESTER: ${{ needs.workflow-setup.outputs.CACHE_KEY_LIBVCX_TESTER }}
+      DOCKER_IMG_NAME_LIBVCX_TESTER: ${{ needs.workflow-setup.outputs.DOCKER_IMG_NAME_LIBVCX_TESTER }}
+      CACHE_KEY_ALPINE_CORE: ${{ needs.workflow-setup.outputs.CACHE_KEY_ALPINE_CORE }}
+      DOCKER_IMG_NAME_ALPINE_CORE: ${{ needs.workflow-setup.outputs.DOCKER_IMG_NAME_ALPINE_CORE }}
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v2
+      - name: Try load from cache.
+        id: cache-image-alpinecore
+        uses: actions/cache@v2
+        with:
+          path: /tmp/imgcache_alpine_core
+          key: ${{ env.CACHE_KEY_ALPINE_CORE }}
+      - name: If alpine_core NOT found in cache, error
+        if: steps.cache-image-alpinecore.outputs.cache-hit != 'true'
+        run: exit -1
+      - name: Load alpine_core image from cache
+        run: |
+          docker load < /tmp/imgcache_alpine_core/img_alpine_core.rar
+          rm /tmp/imgcache_alpine_core/img_alpine_core.rar
+      - name: Try load from cache.
+        id: cache-image-libvcx
```

Comment:
Duplicate cache id (not sure if this may be problematic, but better keep it unique)

Reply:
it's local to the job, nevertheless changed